fc-community-extension/src/serviceWorker.js

```diff
@@ -34,7 +34,12 @@ const mainHandler = async (message, sendResponse) => {
       left: 0,
     });
   } else if (message.type === 'fc-get-notes') {
-    const notes = await getNotes({ postUrl: message.postUrl });
+    const XSupportedNetwork = 'ETHEREUM_SEPOLIA';
+    // Factchain extension only support X on desktop
```

Comment:
I would put this outside of the `serviceWorker`. Instead the X specific code that calls `fc-get-notes` should be passing in the network as argument.
This way when we support Warpcast on desktop it will be neatly split.

Reply:
Btw `fc-rate-note` will need a similar treatment, receiving the network as argument, so that the code in `rateNotes.js` can make sure MM is on the right network.

Reply:
still need to address this

Reply:
TO be addressed in next PR?